app/components/elements/Checkbox.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text } from 'rebass/styled-components';
+import { Checkbox as Base, Label, CheckboxProps } from '@rebass/forms';
+import styled from 'styled-components';
+import cx from 'classnames';
+
+import {
+  colors,
+  fonts,
+  fontSizes,
+  fontWeights } from '../../themes/baseTheme';
+
+const StyledCheckbox = styled(Base)`
+  color: ${colors.border.default};
+  height: 1.5em;
+  width: 1.5em;
+  margin-right: .25em;
```

Comment:
I noticed that the checkbox focus background renders really large due to the default padding on the rebass checkbox.

It can reduced, while maintaining the same general sizing/spacing, by updating the styles as:
```css
  height: 1em;
  width: 1em;
  padding: 0;
  margin-right: 0.5em;
```